src/components/productCardShowMoreButton.tsx

```diff
@@ -1,32 +1,36 @@
 import React from "react";
 import Image from "next/image";
 
-type ProdudctCardImageProps = {
-  showMoreImage: string;
+type ContentProps = {
+  showMoreImage: string[];
   image: string[];
 };
 
+type ProductCardProps = {
+  content: ContentProps;
+  isCardFlipped: boolean;
+  changeImage: (newImage: string) => void;
+  toggleIsFlippedCard: () => void;
+};
+
 export default function productCardShowMoreButton({
   content,
   isCardFlipped,
   changeImage,
-  changeIsCardFlipped,
-}: {
-  content: ProdudctCardImageProps;
-  isCardFlipped: boolean;
-  changeImage: (newImage: string) => void;
-  changeIsCardFlipped: () => void;
-}) {
-  const FrontProductImage = content.image[0];
+  toggleIsFlippedCard,
+}: ProductCardProps) {
+  const frontProductImage = content.image[0];
+  const backProductImage = content.showMoreImage[0];
   return (
     <div className="z-50">
       <button
         className="group-hover:visible invisible flex flex-row absolute top-0 right-0 py-1 px-2 text-xs bg-gray-300"
         onClick={() => {
-          changeIsCardFlipped();
-          isCardFlipped
-            ? changeImage(content.showMoreImage[0])
-            : changeImage(FrontProductImage);
+          toggleIsFlippedCard();
+          const cardToFlipTo = isCardFlipped
+            ? backProductImage
+            : frontProductImage;
+          changeImage(cardToFlipTo);
```

Comment:
This is actually properly a better implementation. Saw it and didn't like it haha
```suggestion
          changeImage(isCardFlipped ? backProductImage : frontProductImage);
```

Reply:
added.